### ISSUE TITLE
disable calculating dynamic bones

### ISSUE THREAD

**Issue by rassi0429** [Yellow-Dog-Man/Resonite-Issues#154]
### Is your feature request related to a problem? Please describe.

There are situations where the FPS is too low(1~3fps), such as in sessions with 30 or more people. Photos may look strange
In this case, DynamicBone's calculation may become incorrect. (Because of acceleration?)

### Describe the solution you'd like

It would be useful to have an option to temporarily not calculate dynamic bones

### Describe alternatives you've considered

Photos can be taken clearly

### Additional Context

If it is an environmental issue and decide not to respond, it is fine to have this issue closed.

**Comment by troyBORG** [comment]
I know on my avatar I have some Flux programed that if that local users FPS goes bellow 25 fps it would turn off my dynamic bones for them locally.  Then once their FPS goes back up above 35 it would turn them back on.

But I agree this should be something that happens automatically for everyone.

**Comment by troyBORG** [comment]
Yeah in some peoples Livestream of Resonite I see people with helicopter ears and they just act like it's normal.  I only see it very rare occasions, but I know most people don't have the newest computers.

**Comment by shiftyscales** [comment]
On #1326 Prime and Frooxius had commented:

> Our current plans are a little different in this area.
> 
> Instead of disabling them at Low FPS we'd rather spend the time optimizing them so that they perform better.
> 
> Due to this it is unlikely that we'll be implementing this specific idea but I will leave the issue open just in case other team members wish to take an alternative angle.
> 
> Thank you!

> I agree with Prime here.
> 
> Disabling them isn't really the approach we would take to resolve the problem.
> 
> As a general rule we try to not "mask" bugs like this - instead our approach would be to resolve the root issue instead.
> 
> In this particular case, the root issue is that they jiggle at low FPS - that is something we want to fix, so it doesn't happen in the first place, but they handle low FPS more gracefully.
> 
> I'm going to close this issue, since this particular course of action isn't something we plan (and we discourage making issues about specific solutions to a problem, rather than a problem itself in our guidelines: https://github.com/Yellow-Dog-Man/Resonite-Issues).
> 
> You're free to open a new issue that focuses on the problem.

As such I am going to rename this issue to focus on stabilizing the physics simulation of dynamic bones under low framerate conditions.
